WhisperAPI/WhisperAPI.Tests/Unit/DocumentFacetsTest.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using Moq.Protected;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using WhisperAPI.Models;
+using WhisperAPI.Services.MLAPI.Facets;
+using WhisperAPI.Tests.Data.Builders;
+
+namespace WhisperAPI.Tests.Unit
+{
+    [TestFixture]
+    public class DocumentFacetsTest
+    {
+        private IDocumentFacets _documentFacets;
+
+        private Mock<HttpMessageHandler> _httpMessageHandlerMock;
+        private HttpClient _httpClient;
+
+        [SetUp]
+        public void Setup()
+        {
+            this._httpMessageHandlerMock = new Mock<HttpMessageHandler>();
+        }
+
+        [Test]
+        public void When_receive_ok_response_from_post_then_returns_result_correctly()
+        {
+            var questions = new List<Question>
+            {
+                QuestionBuilder.Build.Instance
+            };
+
+            const string baseAdress = "http://localhost:5000";
+
+            this._httpClient = new HttpClient(this._httpMessageHandlerMock.Object);
+            this._documentFacets = new DocumentFacets(this._httpClient, baseAdress);
+
+            this._httpMessageHandlerMock.Protected()
```

Comment:
Could be made into a private method for readability

Reply:
Done!